tidb-cloud/monitor-datadog-integration-for-tidb-x.md

```
@@ -34,21 +34,28 @@ Currently, the TiDB Cloud dashboard for <CustomContent plan="essential">{{{ .ess
+    <https://github.com/pingcap/docs/blob/master/tidb-cloud/monitor-datadog-integration-tidb-cloud-dynamic-tracker-essential.json>
+
+    </CustomContent>
+
     <CustomContent plan="premium">
 
     <https://github.com/pingcap/docs/blob/master/tidb-cloud/monitor-datadog-integration-tidb-cloud-dynamic-tracker-premium.json>
 
     </CustomContent>
 
-2. Log in to [Datadog](https://app.datadoghq.com), and then go to **Dashboards** > **Dashboard List**.
+1. Log in to [Datadog](https://app.datadoghq.com), and then go to **Dashboards** > **Dashboard List**.
```

markdownlint-cli2 (0.23.1)

[warning] Ordered list item prefix
Expected: 2; Actual: 1; Style: 1/2/3

(MD029, ol-prefix)